zerver/lib/users.py

```diff
@@ -431,3 +431,13 @@ def get_raw_user_data(realm: Realm, acting_user: UserProfile, client_gravatar: b
                                             custom_profile_field_data = custom_profile_field_data
                                             )
     return result
+
+def get_custom_profile_data_for_user(user_profile: UserProfile) -> Dict[str, Dict[str, Any]]:
```

Comment:
Was this code copy/pasted from some other function?

Reply:
Actually, it was taken from get_raw_user_data.
But I feel that this function could have been avoided.
I am thinking of using get_raw_data itself in notify_reactivated_user.